config/initializers/health_checks.rb

```diff
@@ -0,0 +1,41 @@
+class DelayedJobNeverRanCheck < OkComputer::Check
+  attr_reader :threshold
+
+  def initialize(threshold)
+    @threshold = threshold
+  end
+
+  def check
+    never_ran = Delayed::Job.where('run_at <= ?', 5.minutes.ago).count
```

Comment:
Let's stay with an arbitrary threshold and make that configurable instead of a configurable threshold. _Any_ job that should have run in the past is an indicator that dj is not correctly running. 

It's correct that the check does not have any value as to why it is not correctly running, and we might want to add additional checks to detect long-running blocking jobs.